components/server/src/user/user-controller.ts

```diff
@@ -517,7 +517,7 @@ export class UserController {
             properties: {
                 "auth_provider": user.identities[0].authProviderId,
                 "email": User.getPrimaryEmail(user),
-                "name": User.getName(user),
+                "name": user.identities[0].authName,
```

Comment:
I see how that fixes the particular issue. But I wonder how we're sure that `identities[0]` is what we want? But outside of scope I guess.